sdk/spring-sdk/src/it/java/com/example/wiring/views/UsersByName.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.wiring.views;
+
+import com.example.wiring.valueentities.user.User;
+import com.example.wiring.valueentities.user.UserEntity;
+import kalix.javasdk.view.View;
+import kalix.springsdk.annotations.Query;
+import kalix.springsdk.annotations.Subscribe;
+import kalix.springsdk.annotations.Table;
+import org.springframework.web.bind.annotation.GetMapping;
+import reactor.core.publisher.Flux;
+
+@Table("users_by_name")
+@Subscribe.ValueEntity(UserEntity.class)
+public class UsersByName extends View<User> {
+
+  @GetMapping("/users/by_name/{name}")
+  @Query("SELECT * FROM users_by_name WHERE name = :name")
+  public Flux<User> getUsers(String name) {
```

Comment:
Just crossed my mind that in the future it could make sense to also allow the user to use a Collection instead of Flux.. and in that case it would no longer be a stream response but a wrapped the result (I think we support this correctly as well with a wrapper?)

Reply:
In the gRPC SDKs this is done by using a wrapper, like in: 

```protobuf
message CustomersResponse {
  repeated domain.CustomerState results = 1; 
}
```

What we could do for the Spring SDK, is to generate a synthetic proto message when the method returns a Java collection. The synthetic message will be the wrapper and the users don't need to deal with it. 

The Json response on the side of the caller, will be just a Json Array.

Reply:
Let's keep the ticket current ticket open. We can add this on follow up PRs.